SharedUtilities.js

```diff
@@ -900,12 +1117,13 @@ function logRequest(type, month, year, requestDateTime, windowEndDateTime) {
   }
 
   // Append the new request details
+  // Store dates in ISO 8601 format with explicit timezone offset
   requestLogSheet.appendRow([
     type,
     month,
     year,
-    Utilities.formatDate(requestDateTime, Session.getScriptTimeZone(), 'yyyy-MM-dd HH:mm:ss'),
-    Utilities.formatDate(windowEndDateTime, Session.getScriptTimeZone(), 'yyyy-MM-dd HH:mm:ss'),
+    Utilities.formatDate(requestDateTime, Session.getScriptTimeZone(), "yyyy-MM-dd'T'HH:mm:ssXXX"),
```

Comment:
Most of the code uses getProjectTimeZone (which does Session.getScriptTimeZone, so functionally equivalent,) but this should be standardized.